### ISSUE TITLE
quality of life: tilt ci should report total run time

### ISSUE THREAD

**Issue by adamf** [tilt-dev/tilt#3774]
It'd be nice if tilt ci timed its execution from start to finish and reported it at the end - if we wanted to get fancy maybe also summarize how long each step took to go green. Just a small quality of life improvement, useful for seeing slow spots in ci runs.

**Comment by nicks** [comment]
we're currently working on some hosted infra for helping track of execution times across runs and graphing them!

have you considered using `time tilt ci` to track the time of your tilt ci call? or are you looking for something more sophisticated?

**Comment by adamf** [comment]
Yep, I used `time tilt ci` but it'd be nice to see a summary of step runtimes, etc. Not critical, just nice. :)

**Comment by nicks** [comment]
:+1: 

i like the framing as "how long each step took to go green". i think the logs report docker image build times and pod readiness times, but nothing that connects them all together :thinking:
